backend/config.ts

```diff
@@ -179,6 +179,11 @@ export const apis = {
   bookVideoLink: {
     url: process.env.BVL_URL || 'http://localhost:3000',
   },
+  bookAVideoLinkApi: {
+    enabled: process.env.BOOK_A_VIDEO_LINK_API_ENABLED === 'true',
+    url: process.env.BOOK_A_VIDEO_LINK_API_URL || 'http://localhost:8083/',
+    timeoutSeconds: toNumber(process.env.BOOK_A_VIDEO_LINK_API_TIMEOUT_SECONDS) || 30,
+  },
```

Comment:
I see that the old BVL has a url above this .. any switch required to recognise that the new one is alive? Worth a comment here? Maybe its covered below ..